File: src/supports/katas/roman-numerals/java/RomanTest.java
// RomanTest.java — Kata Roman Numerals — Tests de démarrage
// Le premier test est écrit pour vous. Suivez la méthode Ping-Pong TDD !

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class RomanTest {

    // 🏓 Test 1 — Le premier est cadeau !
    @Test
    void test_1_returns_I() {
        assertEquals("I", Roman.toRoman(1));
    }

    @Test
    void test_2_returns_II() {
        assertEquals("II", Roman.toRoman(2));
    }
    @Test
    void test_3_returns_III() {
        assertEquals("III", Roman.toRoman(3));
    }
    @Test
    void test_4_returns_IV() {
        assertEquals("IV", Roman.toRoman(4));
    }
    @Test
    void test_5_returns_V() {
        assertEquals("V", Roman.toRoman(5));
    }


    //   Roman.toRoman(4) -> "IV"
    //   Roman.toRoman(5) -> "V"
    //   Roman.toRoman(6) -> "VI"
    //   Roman.toRoman(9) -> "IX"
    //   Roman.toRoman(10) -> "X"
    //   Roman.toRoman(14) -> "XIV"
    //   Roman.toRoman(40) -> "XL"
    //   Roman.toRoman(50) -> "L"
    //   Roman.toRoman(90) -> "XC"
    //   Roman.toRoman(100) -> "C"
    //   Roman.toRoman(400) -> "CD"
    //   Roman.toRoman(500) -> "D"
    //   Roman.toRoman(900) -> "CM"
    //   Roman.toRoman(1000) -> "M"
    //   Roman.toRoman(2024) -> "MMXXIV"
    //   Roman.toRoman(3999) -> "MMMCMXCIX"
}

File: src/supports/katas/roman-numerals/javascript/roman.test.js
// roman.test.js — Kata Roman Numerals — Tests de démarrage
// Le premier test est écrit pour vous. Suivez la méthode Ping-Pong TDD !

const { toRoman } = require("./roman");

// 🏓 Test 1 — Le premier est cadeau !
test("1 returns I", () => {
    expect(toRoman(1)).toBe("I");
});

// 🏓 À vous de continuer en Ping-Pong TDD !
// Progression suggérée :
//   toRoman(2) -> "II"
//   toRoman(3) -> "III"
//   toRoman(4) -> "IV"
//   toRoman(5) -> "V"
//   toRoman(6) -> "VI"
//   toRoman(9) -> "IX"
//   toRoman(10) -> "X"
//   toRoman(14) -> "XIV"
//   toRoman(40) -> "XL"
//   toRoman(50) -> "L"
//   toRoman(90) -> "XC"
//   toRoman(100) -> "C"
//   toRoman(400) -> "CD"
//   toRoman(500) -> "D"
//   toRoman(900) -> "CM"
//   toRoman(1000) -> "M"
//   toRoman(2024) -> "MMXXIV"
//   toRoman(3999) -> "MMMCMXCIX"

